cv_next/server/api/firebaseHelper.ts

```diff
@@ -1,25 +1,25 @@
 import 'server-only'
 
-import { FirebaseApp, initializeApp } from "firebase/app";
+import { App, initializeApp, getApp, getApps} from "firebase-admin/app";
+import { createRequire } from 'module'
+const require = createRequire(import.meta.url)
+const cvSdk = require("@/cv-sdk.json");
```

Comment:
This is some weird workaround. Cv-sdk is a rename for the json file for authenticating the app. I needed to import it directly to the program, and in my tests, "require" did the job, unlike "import". If this is crucial, I'll check again.

Reply:
You mean the authentication settings that are found in `firebase.config.js`? Why would you rename it?
Also yes, please try to use import, consistency is important. If there's an actual problem that prevents you from using `import`, let me know and use `require` instead, but I don't see a reason why that'd happen.

Reply:
The authorizing key for admin is different from the regular. But it's ok, I'll change the definitions file so it'll fit with the original route of process env variables.

Reply:
Oh man, this is the main reason I wanted this package, precisely because this is wrong in our case: "Besides being snappy fast, it will provide better UX as the same page will always produce the same results, unlike pagination by offset where items constantly shift." - for changing data (e.g. cv can be deleted), fetching by last doc id will _not_ always produce the same result. This is way for UX reasons, offset is better.
But if firebase doesn't allow us to query offset efficiently, I'll just keep on hating firebase and we'll keep the original implementation.

Reply:
I apologize for not catching this sooner.. ooof

Reply:
Oh and another thing my dumbass (pardon my French) found out last night - firestore security rules don't apply on Firebase Admin.

Reply:
Wait. Are we using this feature? Does that mean we can't use Admin?
Why can't they make a proper package ffs

Reply:
TBH after all this time, and thinking about how we'd like to authenticate users and have security rules that matter, I've come to the conclusion that we should backtrack on replacing the firebase library. I apologize for making quite a mess, that didn't really prove itself in any way. Is there anything I could do now?